sec-priv-q.md

```diff
@@ -0,0 +1,303 @@
+acceptable mechanism.  Differential privacy mechanisms also require choosing
+$\epsilon$ and maybe $\delta$ values.  These values determine the amount of
+noise that is added and the rate at which sites are able to accumulate
+information.  No firm conclusions have been reached on this point, though we
+have a tentative design that would allow different user agents to set different
+values for these parameters.
+
+Even with differential privacy, sites might gain more information about users
+over longer periods.  We believe this to be a tolerable privacy loss as it is
+only sites that have long term relationships with users that are able to obtain
+additional information in this way.  A strict cap on information release would
+render the system completely useless for attribution purposes.  The only
+recourse we are aware of is to choose parameters that are more conservative to
+reduce the overall rate at which sites can obtain this information.  This needs
+to be balanced against the potential adverse effect that added noise has on how
+useful the information is for performing attribution.
+
+We are seeking to gather more information before making decisions regarding
+differential privacy mechanisms and parameters.
+
+### How do the features in your specification deal with personal information,
+    personally-identifiable information (PII), or information derived from them?
+
+The IPA API provides pseudo-identifiers for individuals and makes those
+identifiers available for use in an MPC system.  These identifiers are protected
+by secret sharing, encryption, and a system of contracts that prevent even the
+MPC operators from accessing values.
+
+### How do the features in your specification deal with sensitive information?
+
+See previous answers.
+
+### Do the features in your specification introduce new state for an origin that
+    persists across browsing sessions?
+
+Yes.  Though the explainer does not deal with this in great detail, our intent
+is to ensure that state clearing events, such as clearing cookies, will result
+in the corresponding state being cleared.
+
+Two factors complicate this.  The API provides information that is inherently
+cross-site and the epoch design relies on values that are stable for an entire
+epoch (a week).  This makes state-clearing events, particularly those that are
+narrowly targeted, somewhat challenging to reason about.  We haven't worked
+through all of the requirements and constraints in this area, such that we might
+infer the correct design.
+
+### Do the features in your specification expose information about the
+    underlying platform to origins?
+
+No.  However, if a compatible system is developed in the underlying platform for
+the management of match keys, user agent implementations might move to unify the
+two sources of match keys.  This would enable attribution that crosses from the
+web into applications.  No such mechanism currently exists.
```

Comment:
well... it kind of does with PCM: https://webkit.org/blog/12042/pcm-for-in-app-direct-response-advertising/

Reply:
I'm talking about a system that is compatible with IPA.  I don't think PCM really does that.  An analogous system exists, sure.